srcpkgs/openbsd-file/patches/no-x86-seccomp.patch

```diff
@@ -0,0 +1,17 @@
+--- seccomp-sandbox.c.orig	2020-01-30 15:28:45.061751084 +0100
++++ seccomp-sandbox.c	2020-01-30 15:34:05.935174237 +0100
+@@ -40,6 +40,14 @@
+ #define SECCOMP_AUDIT_ARCH AUDIT_ARCH_I386
+ #elif defined __x86_64__ || defined __amd64__
+ #define SECCOMP_AUDIT_ARCH AUDIT_ARCH_X86_64
++#elif defined __arm__
++#define SECCOMP_AUDIT_ARCH AUDIT_ARCH_ARM
++#elif defined __aarch64__
++#define SECCOMP_AUDIT_ARCH AUDIT_ARCH_AARCH64
++#elif defined __ppc__
```

Comment:
This is incorrect. The logic should be:

```
#elif defined(__powerpc64__) && defined(__LITTLE_ENDIAN__)
#define SECCOMP_AUDIT_ARCH AUDIT_ARCH_PPC64LE
#elif defined(__powerpc64__)
#define SECCOMP_AUDIT_ARCH AUDIT_ARCH_PPC64
#elif defined(__powerpc__)
#define SECCOMP_AUDIT_ARCH AUDIT_ARCH_PPC
```

The order is important as `__powerpc__` is also defined on ppc64. There is no `__ppc__` or `__ppc64__`.

Reply:
Ok, wasn't sure what was the proper symbol